### PR TITLE
documented how to create conditional links

### PR DESCRIPTION
**Pull request by darrenrichmond**
This PR closes #[insert associated issue number here]

#### What does this PR do?
doc how to create conditional links (like issue 171)

#### How does this PR make you feel? [:link:](http://giphy.com/categories/emotions/)

http://i.giphy.com/l46Cl6JOKu0fbiR3O.gif

### CODE REVIEW

File: DEVDOCS.md [hunk 1]
```diff
@@ -32,7 +32,57 @@ We use Redux to ensure state transitions are understandable. This happens by man
 ###State Transition Example
 Let's walk through an example of the first step of our primary use case: an Organizer registering on the Landing page and going to the CreateEvent page.
 We suggest that you have the app installed on your machine as described on the [README](README.md), and run through the app with your dev tools window open,
-so you can verify for yourself what's going on.
+so you can verify for yourself what's going on. [TBC]
```
Comment (wdoug) [inline DEVDOCS.md]: TBC? "To be continued"?

File: DEVDOCS.md [hunk 2]
```diff
@@ -32,7 +32,57 @@ We use Redux to ensure state transitions are understandable. This happens by man
...
 
+##Quick Tips
+Here are a few tips on how to accomplish specific goals
 
+###Conditional Hard-Coded Links
+Say you want a peice of text in the UI to be a link to another page, but which page is linked is dependent on some application state.
```
Comment (wdoug) [inline DEVDOCS.md]: `peice` -> `piece`

File: DEVDOCS.md [hunk 3]
```diff
@@ -32,7 +32,57 @@ We use Redux to ensure state transitions are understandable. This happens by man
...
+```JavaScript
+import { isLoggedIn, getUser, isMusician } from '../reducers/userReducer';
+```
+
+2. make sure your container maps the proper state to your prop
+```JavaScript
+const mapStateToProps = (state) => {
+  return {
+    isLoggedIn: isLoggedIn(state),
+    user: getUser(state),
```
Comment (wdoug) [inline DEVDOCS.md]: For documentation purposes, we should probably leave out unnecessary info (e.g. `isLoggedIn` and `user` here)

File: DEVDOCS.md [hunk 4]
```diff
@@ -32,7 +32,57 @@ We use Redux to ensure state transitions are understandable. This happens by man
...
+      <Route path="/event/:id" component={EventContainer} />
+      <Route path="/musicianProfile" component={MusicianProfile} />
+      <Route path="/eventsAttending" component={EventsAttendingContainer} />
+      <Route path="/musician/:id" component={MusicianContainer} />
+</Route>
+```
+
+5. In your component, set up a conditional that links to the correct page based on the prop
+```JavaScript
+  Hello, <Link to={this.props.isMusician ? '/musicianProfile' : 'organizerProfile'}>{this.props.user.email}</Link>
```
Comment (wdoug) [inline DEVDOCS.md]: `'organizerProfile'` -> `'/organizerProfile'`

File: DEVDOCS.md [hunk 5]
```diff
@@ -32,7 +32,57 @@ We use Redux to ensure state transitions are understandable. This happens by man
...
+    isLoggedIn: PropTypes.bool.isRequired,
+    logoutUser: PropTypes.func.isRequired,
+    user: PropTypes.object,
+    isMusician: PropTypes.bool.isRequired
+  };
+```
+
+4. Find your target pages in routes.js
+```JavaScript
+<Route component={AuthenticatedRoutesContainer} >
```
Comment (wdoug) [inline DEVDOCS.md]: Do we need to include this step? At least it seems like the code example is unnecessary as the user isn't going to change anything there.

File: DEVDOCS.md [hunk 6]
```diff
@@ -32,7 +32,57 @@ We use Redux to ensure state transitions are understandable. This happens by man
...
+  };
+};
+```
+
+3. In your component, require your prop
+```JavaScript
+static propTypes = {
+    isLoggedIn: PropTypes.bool.isRequired,
+    logoutUser: PropTypes.func.isRequired,
+    user: PropTypes.object,
```
Comment (wdoug) [inline DEVDOCS.md]: No need to include these properties in this example.

File: DEVDOCS.md (hunk 3 above)
Comment (darrenrichmond) [reply DEVDOCS.md]: fixed

File: DEVDOCS.md (hunk 6 above)
Comment (darrenrichmond) [reply DEVDOCS.md]: fixed

File: DEVDOCS.md (hunk 5 above)
Comment (darrenrichmond) [reply DEVDOCS.md]: fixed

File: DEVDOCS.md (hunk 4 above)
Comment (darrenrichmond) [reply DEVDOCS.md]: fixed

**Pull request merged**